src/main/java/org/craftercms/studio/impl/v2/security/SemanticsAvailableActionsResolverImpl.java

```diff
@@ -84,6 +92,20 @@ private long applySpecialUseCaseFilters(String username, String siteId, Item ite
             throws ServiceLayerException, UserNotFoundException {
         long result = availableActions;
 
+        if (StringUtils.equals(item.getPath(), HOME_PAGE_PATH)) {
```

Comment:
Why isn't this a pattern like the others?

Reply:
hardcoded home page ... it is always on the same path and does not need to be configured. If configured, somebody can remove protection by accident or intentionally